src/cli/cli.cc

```diff
@@ -370,6 +370,71 @@ static String getCxxFlags() {
   return flags.size() > 0 ? " " + flags : "";
 }
 
+static Map getConfig (fs::path p) {
+  std::stringstream compileConfigCommand;
+
+  if (getEnv("CXX").size() == 0) {
+    log("warning! $CXX env var not set, assuming defaults");
+
+    if (platform.win) {
+      setEnv("CXX=clang++");
+    } else {
+      setEnv("CXX=/usr/bin/clang++");
+    }
+  }
+
+  compileConfigCommand
+    << getEnv("CXX")
+    << " -std=c++2a -v"
+    << " -I" << prefixFile()
+    << " -I" << prefixFile("src/config.hh")
+    << " -I" << p
+    << " -I" << fs::current_path()
+    << " " << prefixFile("src/config.cc")
+    << " -o " << prefixFile("bin/ssc-config")
```

Comment:
maybe this should build to a local dot directory (`.ssc/tmp`) since the files where `prefixFile()` looks is global in the file system

Reply:
I think we should open a separate issue for this, I like the idea. There are some other files that we may need in the future. This is probably a part of a broader conversation.